src/RMKERR.CPP

```diff
@@ -1077,7 +1077,7 @@ RC DbFileOpen(
 	if (_dbFName)
 		unlink( _dbFName);		// erase file -- don't garbage up directory
 #else
-	static char dbFName[ _MAX_PATH] = { 0 };
+	static char dbFName[FILENAME_MAX] = { 0 };
```

Comment:
Is it correct for some FILENAME_MAX to still be used, or should this also be CSE_MAX_PATH?

Reply:
I think I forgot to fix the comments that's why there were some missing FILENAME_MAX. They should all be replaced.